packages/compass-components/src/components/accordion.tsx

```diff
@@ -5,13 +5,9 @@ import Icon from '@leafygreen-ui/icon';
 import { uiColors } from '@leafygreen-ui/palette';
 import { useId } from '@react-aria/utils';
 
-const labelStyles = css({
-  padding: 0,
-  margin: 0,
-  fontWeight: 'bold',
-});
-
 const buttonStyles = css({
+  fontWeight: 'bold',
+  fontSize: 14,
```

Comment:
How can we centralize this?

Reply:
Added a `compass-font-sizes` file to `compass-components` with this in it. Updated the other places in the connect form to use it as well. It's definitely something only custom components should really be using, but I think at least having it in one place that things reference will keep things more manageable.